src/components/calcite-block/calcite-block.scss

```diff
@@ -79,7 +79,7 @@ calcite-handle {
 .header .title .heading {
   @apply p-0
     text--1
-    text-color-3
+    text-color-1
```

Comment:
This is the default color for the heading and as per @bstifle's suggestion, we made it text-1 [when the Block is open](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-block/calcite-block.scss#L152).

Reply:
hey i think we are both wrong. We should match what Accordion is doing:

ui-text-2 idle and closed

ui-text-1 hover and open

Reply:
@bstifle. Looks like the open state is already there, so you just need to the default state to change to `text-color-2`, correct?

Reply:
Yeah. That would keep the open/close accordion type components cohesive. You good with that @asangma ?

Reply:
I think the difference between 1 and 2 is too subtle...especially in more complex interfaces.
Might be fine on an awesome monitor with good eyes, but not so great in other cases.

Reply:
I'll defer, but here's a comparison.

![image](https://user-images.githubusercontent.com/12503298/128735430-f7dc345d-9eae-4cf2-85b8-ddd578038293.png)

Reply:
it's pretty weird to have the heading and summary the same color. we dont do that anywhere else

Reply:
Copy that.